assets/erc-7291/contracts/IPBMRC1_TokenReceiver.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Same license concern.

Reply:
Licenses of sample code have been changed to CC0-1.0